globus_sdk/transfer/client.py

```diff
@@ -1772,6 +1821,52 @@ def endpoint_manager_task_successful_transfers(
             paging_style=PaginatedResource.PAGING_STYLE_MARKER,
         )
 
+    def endpoint_manager_task_skipped_errors(self, task_id, num_results=100, **params):
+        """
+        Get skipped errors for a completed Task as an admin.
+
+        ``GET /endpoint_manager/task/<task_id>/skipped_errors``
+
+        :param task_id: The ID of the task to inspect.
+        :type task_id: str
+        :param num_results: The number of skipped error records to fetch from the
+            service. May be set to ``None`` to request the maximum allowable number of
+            results. [Default: ``100``]
+        :type num_results: int or None, optional
+        :param params: Any additional parameters will be passed through as query params.
+        :type params: dict, optional
+        :rtype: :class:`PaginatedResource
+                <globus_sdk.transfer.paging.PaginatedResource>`,
+                an iterable of :class:`GlobusResponse
+                <globus_sdk.response.GlobusResponse>`
+
+        **External Documentation**
+
+        See
+        `Get task skipped errors as admin\
+        <https://docs.globus.org/api/transfer/advanced_endpoint_management/#get_task_skipped_errors_as_admin>`_
+        in the REST documentation for details.
+        """
+        task_id = safe_stringify(task_id)
+        self.logger.info(
+            (
+                "TransferClient.endpoint_manager_task_"
+                "skipped_errors({}, ...)".format(task_id)
+            )
+        )
```

Comment:
This doesn't really matter, but may I suggest
```python
        self.logger.info(
            "TransferClient."
            "endpoint_manager_task_skipped_errors({}, ...)".format(task_id)
        )
```
as an alternative? The long-line rule may demand that we break things up, but I think the above is more readable because it avoids breaking a "long word".

Reply:
Agreed - I think this might have been auto-formatted since I tend to err in the direction of too long a line ^^"